components/primary_button/sample.html

```diff
@@ -5,6 +5,26 @@
     <link rel="stylesheet" href="./style.css">
     <link href="https://fonts.googleapis.com/css?family=Noto+Sans" rel="stylesheet">
     <title>Primary Button</title>
+    <style>
+      .BC-Gov-PrimaryButton {
```

Comment:
We should come up with a convention around class names and determine whether or not we want to use `bc-gov` or similar in sample classnames.  There pros and cons to each approach.  I've opened a couple of issues related to this.